app/src/main/java/org/schabi/newpipe/info_list/dialog/InfoItemDialog.java

```diff
@@ -290,18 +289,15 @@ public Builder addMarkAsWatchedEntryIfNeeded() {
                     && infoItem.getStreamType() != StreamType.AUDIO_LIVE_STREAM) {
                 addEntry(StreamDialogDefaultEntry.MARK_AS_WATCHED);
             }
-            return this;
         }
 
         /**
-         * Adds the {@link StreamDialogDefaultEntry.PLAY_WITH_KODI} entry if it is needed.
-         * @return the current {@link Builder} instance
+         * Adds the {@link StreamDialogDefaultEntry#PLAY_WITH_KODI} entry if it is needed.
          */
-        public Builder addPlayWithKodiEntryIfNeeded() {
+        public void addPlayWithKodiEntryIfNeeded() {
```

Comment:
This is not how the Builder pattern works (also above)

Reply:
As far as I can tell, it doesn't need to be a Builder method though, as the Builder isn't being modified here. Am I just seeing it wrong?

Edit: Nevermind, I am just smooth-brained lol. Will correct.